pipeline/merger/src/main/scala/weco/pipeline/merger/services/Merger.scala

```diff
@@ -176,14 +177,26 @@ object Merger {
   }
 }
 
-object PlatformMerger extends Merger {
-  import weco.catalogue.internal_model.image.ParentWork._
+object DefaultPlatformMerger extends BasePlatformMerger {
+  override val targetPrecedence: BaseTargetPrecedence = DefaultTargetPrecedence
+  override val passThroughTransform: PassThroughTransform =
+    DefaultPassThroughTransform$
+}
+
+object TeiPlatformMerger extends BasePlatformMerger {
+  override val targetPrecedence: BaseTargetPrecedence = TeiTargetPrecedence
+  override val passThroughTransform: PassThroughTransform =
+    TeiPassThroughTransform$
+}
+trait BasePlatformMerger extends Merger {
```

Comment:
What's the difference between `BasePlatformMerger` and `Merger`?

Reply:
I think in this instance, it is just applying the same split in inheritance as everywhere else.

I think in this case Merger is actually just a set type class ops - I'll see if I can make this more clear.